sdcm/sct_config.py

```diff
@@ -1534,6 +1556,10 @@ def verify_configuration(self):
         self._check_per_backend_required_values(backend)
         if backend in ['aws']:
             self._check_aws_multi_region_params()
+
+        # verify that correct configuration set for using ebs-storage
```

Comment:
No need to have a comment here, function name is self-explanatory

Reply:
fixed